common/src/Assets/Texture.h

```diff
@@ -46,7 +46,9 @@ namespace TrenchBroom {
 
             size_t m_usageCount;
             bool m_overridden;
-            
+
+            GLenum m_format = GL_RGB;
```

Comment:
Incidentally, the windows build failed because of this.

> C:\projects\trenchbroom\common\src\Assets/Texture.h(50): error C2864: 'TrenchBroom::Assets::Texture::m_format' : only static const integral data members can be initialized within a class [C:\projects\trenchbroom\cmakebuild\common.vcxproj]